### PR TITLE
SB : Sample Disable queue from administration client 

### PR DESCRIPTION
**Pull request by hemanttanwar**
We got user query on how to enable disable Service bus entity.https://github.com/Azure/azure-sdk-for-java/issues/18214

So created this sample .

This sample shows how to update queue using Administration client.

### CODE REVIEW

File: sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+        // "<<queue-name>>" will be the name of the Service Bus queue instance you created
+        // inside the Service Bus namespace.
+        QueueProperties queueProperties = administrationClient.getQueue("<<queue-name>>");
+
+        System.out.printf("Before update queue properties status :[%s], Max Delivery count :[%d]. %n" ,
+            queueProperties.getStatus(), queueProperties.getMaxDeliveryCount());
+
+        // You can update 'QueueProperties' object with properties you want to change.
+        queueProperties.setStatus(EntityStatus.DISABLED);
```
Comment (conniey) [inline sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java:43]: This is a fluent model, you can chain these properties.

File: sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            + "SharedAccessKey={key}";
+
+        // Create a administrator client using connection string.
+        ServiceBusAdministrationClient administrationClient = new ServiceBusAdministrationClientBuilder()
+            .connectionString(connectionString)
+            .buildClient();
+
+        // "<<queue-name>>" will be the name of the Service Bus queue instance you created
+        // inside the Service Bus namespace.
+        QueueProperties queueProperties = administrationClient.getQueue("<<queue-name>>");
```
Comment (conniey) [inline sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java:37]: The variable names are really wordy. I think `properties`, `client`, `updatedProperties` versus `queueProperties` and `administrationClient` is enough.

File: sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+        System.out.printf("Before update queue properties status :[%s], Max Delivery count :[%d]. %n" ,
```
Comment (conniey) [inline sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java:39]: The spacing is off on this sentence. "Before queue properties status: [%s], max delivery count: [%d]. %n"

There is one space after a colon and no space before.

File: sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+            queueProperties.getStatus(), queueProperties.getMaxDeliveryCount());
+
+        // You can update 'QueueProperties' object with properties you want to change.
+        queueProperties.setStatus(EntityStatus.DISABLED);
+        queueProperties.setMaxDeliveryCount(9);
+
+        QueueProperties updatedQueueProperties = administrationClient.updateQueue(queueProperties);
+
+        System.out.printf("After update queue properties status :[%s], Max Delivery count :[%d]. %n" ,
```
Comment (conniey) [inline sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java:48]: Same here. "After queue properties status: [%s], max delivery count: [%d].%n"

File: sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+     * @param args Unused arguments to the program.
+     * @throws InterruptedException If the program is unable to sleep while waiting for the receive to complete.
+     */
+    public static void main(String[] args) throws InterruptedException {
+        // The connection string value can be obtained by:
+        // 1. Going to your Service Bus namespace in Azure Portal.
+        // 2. Go to "Shared access policies"
+        // 3. Copy the connection string for the "RootManageSharedAccessKey" policy.
+
+        String connectionString = "Endpoint={fully-qualified-namespace};SharedAccessKeyName={policy-name};"
```
Comment (YijunXieMS) [inline sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java:27]: Same as what you're doing right now for other samples, use environment variable so this sample can be run without any changes?

File: sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+ * Sample example showing how to update queue properties in Service Bus Queue.
+ */
+public class AdministrationClientUpdateQueueSample {
+    /**
+     * Main method to how update queue properties in Service Bus Queue.
+     *
+     * @param args Unused arguments to the program.
+     * @throws InterruptedException If the program is unable to sleep while waiting for the receive to complete.
+     */
+    public static void main(String[] args) throws InterruptedException {
```
Comment (srnagar) [inline sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java:21]: Why does this throw `InterruptedException`?

File: sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+        // "<<queue-name>>" will be the name of the Service Bus queue instance you created
+        // inside the Service Bus namespace.
+        QueueProperties properties = client.getQueue("<<queue-name>>");
+
+        System.out.printf("Before queue properties status: [%s], Max Delivery count: [%d].%n",
+            properties.getStatus(), properties.getMaxDeliveryCount());
+
+        // You can update 'QueueProperties' object with properties you want to change.
+        properties.setStatus(EntityStatus.DISABLED).setMaxDeliveryCount(9);
```
Comment (YijunXieMS) [inline sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java:43]: It's atypical to update entity status. I suggest use other properties mentioned here https://docs.microsoft.com/en-us/rest/api/servicebus/update-queue

**Review by srnagar — APPROVED** [review]

**Review by YijunXieMS — APPROVED** [review]

File: sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java (hunk 5 above)
Comment (hemanttanwar) [reply sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java:27]: I have another PR where I am converting samples which will use env variables and also setup  pipeline to create those resources and run them. This work would have to be don in that PR https://github.com/Azure/azure-sdk-for-java/pull/18185

File: sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java (hunk 7 above)
Comment (hemanttanwar) [reply sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java:43]: We did had a user query,  but I agree that it is not regular use case to disable the queue.

File: sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+/**
+ * Sample example showing how to update queue properties in Service Bus Queue.
+ */
+public class AdministrationClientUpdateQueueSample {
+    /**
+     * Main method to how update queue properties in Service Bus Queue.
+     *
+     * @param args Unused arguments to the program.
+     * @throws InterruptedException If the program is unable to sleep while waiting for the receive to complete.
```
Comment (srnagar) [inline sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java:20]: This should be removed as well

File: sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java [hunk 9]
```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (conniey) [inline sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java:1]: This should be added to the samples README too.

File: sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+import com.azure.messaging.servicebus.administration.models.QueueProperties;
+
+import java.time.Duration;
+
+/**
+ * Sample example showing how to update properties for Service Bus Queue.
+ */
+public class AdministrationClientUpdateQueueSample {
+    /**
+     * Main method to show how update properties for Service Bus Queue.
```
Comment (srnagar) [inline sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/AdministrationClientUpdateQueueSample.java:17]:
```suggestion
     * Main method to show how to update properties of Service Bus Queue.
```

**Pull request merged**